tests/unit_tests/libs_tests/SFML_tests/CMakeLists.txt

```diff
@@ -13,13 +10,8 @@ set(TEST_SRC_ROOT
     ${SRC_ROOT}/GraphicsTextComponentTests.cpp
 )
 
-## Documented Example ##
 create_test(
-    ## First, the name of the test ##
```

Comment:
And Last

Reply:
IT's the template documentation. She has nothing to do here